views/install.sh.erb

```diff
@@ -439,8 +447,13 @@ install_file() {
   echo "Installing $project $version"
   case "$1" in
     "rpm")
-      echo "installing with rpm..."
-      rpm -Uvh --oldpackage --replacepkgs "$2"
+      if test "x$platform" = "xnexus" || test "x$platform" = "xios_xr"; then
+        echo "installing with yum..."
+        yum install -yv "$2"
```

Comment:
To your original point Lamont, it IS an omnibus binary, and there are no repos set up in yum anyway so yum couldn't find any deps if it tried.